tools/rw-heatmaps/rw-benchmark.sh

```diff
@@ -170,6 +170,8 @@ for RATIO_STR in ${RATIO_LIST}; do
       CURRENT_ETCD_PID=$!
       sleep 5
 
+      "${ETCD_BIN_DIR}"/resource_monitor.sh --pid "${CURRENT_ETCD_PID}" --output-file "${OUTPUT_FILE/%.csv/}-resources-ratio-${RATIO}-conn-${CONN_CLI_COUNT}-value-${VALUE_SIZE}.csv" &
```

Comment:
question: is this reusing https://github.com/ahrtr/resource-monitor/blob/main/resource_monitor.sh directly? If yes, does it mean that we need to manually that it's present under the /bin directory?

Reply:
My bad, this change shouldn't be here. I'll remove it from the commit.

Reply:
When rebasing I added a commit that didn't belong to this branch. I removed it.